### ISSUE TITLE
[BUG] megaport_vxc changes b_end.product_uid on import

### ISSUE THREAD

**Issue by maksym-dilanian** [megaport/terraform-provider-megaport#177]
**Describe the bug**
VXC was created manually and now I`m importing it into Terraform state.  
Below is a screenshot of VXC:
![image](https://github.com/user-attachments/assets/87780394-422f-43af-b53a-52edfdda2879)

The product is sourced from `megaport_partner` data source:
```tf
data "megaport_location" "coresite_va1" {
  name = "CoreSite VA1"
}

data "megaport_partner" "aws_port" {
  company_name = "AWS"
  connect_type = "AWSHC"
  product_name = "US East (N. Virginia) (us-east-1)"
  diversity_zone = "red"
  location_id  = data.megaport_location.coresite_va1.id
}
```

Terraform plan shows that `current_product_uid = "a82d4bda-37ca-4156-b60b-12fba2a6fd6e"` but `requested_product_uid` will be changed to `"6dab69c3-d19b-40cc-9aab-ba959fd5009f"`. Also `location` and `product_name` will be changed.

```tf
   # megaport_vxc.aws_vxc_01 will be updated in-place
   # (imported from "3d67ea80-0612-4886-a290-c721655fe6f3")
   ~ resource "megaport_vxc" "aws_vxc_01" {
        ~ b_end                = {
           ~ current_product_uid   = "a82d4bda-37ca-4156-b60b-12fba2a6fd6e" -> (known after apply)
           ~ location              = "CoreSite VA1" -> (known after apply)
           ~ location_id           = 68 -> (known after apply)
           + ordered_vlan          = (known after apply)
             owner_uid             = "605cb850-dfb4-4a05-a171-8bf17757b3a2"
           ~ product_name          = "US East (N. Virginia) (us-east-1)" -> (known after apply)
           ~ requested_product_uid = "" -> "6dab69c3-d19b-40cc-9aab-ba959fd5009f"
         }
```
 
**Terraform and Provider Version**
```
 + terraform version
 Terraform v1.9.6
 on linux_amd64
 + provider registry.terraform.io/megaport/megaport v1.2.0
```

**Terraform File and Steps to Reproduce**

[megaport_tf.txt](https://github.com/user-attachments/files/17599002/megaport_tf.txt)
[plan.txt](https://github.com/user-attachments/files/17599003/plan.txt)

**Additional context**
Cannot even find to which product uid `a82d4bda-37ca-4156-b60b-12fba2a6fd6e` corresponding to.
If I source all AWS products, this id is not present in the list:
```tf
data "megaport_partner" "all" {
  company_name = "AWS"
}

output "megaport_partner_all" {
  value = data.megaport_partner.all
}
```

**Comment by mega-alex** [comment]
Hi there, this should be addressed with the latest `1.2.4` release, mind giving it a shot? Essentially we should now be ignoring that attribute on that end of a cloud VXC, and we'll put a warning in the log about how it won't be changed. This is a little cumbersome, but you can't move a CSP end of a VXC anyway so it shouldn't impact functionality. 

To give some context as to why this was an issue in the first place, the Megaport API will pick a different port than provided by the user (but in the same onramp) so we can do capacity management. This isn't an issue for the portal or most API users, but terraform is particular about the state not changing from the user plan, and this is why we have `requested_product_uid` and `current_product_uid` in the first place.

Also as for why `product_name`, `location`, etc. are all "known after apply" it is because you can move a VXC. Because moving it will change those dependent attributes, the provider can't know ahead of time that those will/will not change.

**Comment by maksym-dilanian** [comment]
The plan has changed:
```tf
       ~ b_end = {
           ~ current_product_uid   = "a82d4bda-37ca-4156-b60b-12fba2a6fd6e" -> (known after apply)
           + inner_vlan            = (known after apply)
           ~ location              = "CoreSite VA1" -> (known after apply)
           ~ location_id           = 68 -> (known after apply)
           + ordered_vlan          = (known after apply)
             owner_uid             = "605cb850-dfb4-4a05-a171-8bf17757b3a2"
           ~ product_name          = "US East (N. Virginia) (us-east-1)" -> (known after apply)
             requested_product_uid = ""
             secondary_name        = ""
           ~ vlan                  = 1472 -> (known after apply)
             vnic_index            = 0
         }
```

And a warning is displayed:
```
Warning: VXC B-End product UID is from a partner port, therefore it will not be changed.

  with megaport_vxc.aws_vxc_01,
  on resourses.tf line 168, in resource "megaport_vxc" "aws_vxc_01":
 168: resource "megaport_vxc" "aws_vxc_01" {

VXC B-End product UID is from a CSP partner port, therefore it will not be
changed.
```
